content/foundations/color/overview.mdx

```diff
@@ -0,0 +1,148 @@
+  Color is a fundamental part of Primer's visual language. In this guide you will learn about the principles,
+  patterns, and abstractions that make up Primer’s color system.
+---
+
+import {Box} from '@primer/react'
+
+## Overview
+
+<img width="960" alt="wip" src="https://github.com/primer/primitives/assets/18661030/615e3e06-9289-4adc-ab85-2213fae96e8c" style="margin-bottom: 1rem;" />
+
+GitHub's UI offers two color modes, `light` and `dark`. Primer supports both color modes across nine [themes](../../primitives/getting-started#theming). Every pattern in Primer is built to work across all color modes out of the box.
+
+### Color modes in Figma
+
+When designing product interfaces in Figma, we recommend using **light mode** or **dark mode** as the Primer Web Figma library provides components and tokens in those two modes.
+You can learn more about using design tokens in Figma in our [Figma guidelines](../../guides/figma/getting-started#styles-and-variables).
+
+### Color design tokens
+
+Primer delivers colors in the form of [design tokens](../../primitives/getting-started). Design tokens are a layer of abstraction that allows for better maintainability, consistency and theming. Within Primer, design tokens are represented as CSS variables for code and Figma variables for design. For a full reference table of all available design tokens, see the [color CSS variables](../../primitives/color) page.
+
+As an example of how color tokens work, when `bgColor-default` is referenced for a background color, the value of that token will automatically change depending on the color mode.
+
+<img
+  width="960"
+  alt="Same component shown in light mode and dark mode color palettes"
+  src="https://github.com/primer/react/assets/18661030/21726ed0-2175-44a3-a821-af002778c0b3"
+/>
+
+### Design token categories
+
+Primer design tokens are categorized into three groups: **base**, **functional**, and **component/pattern**. To read more about the naming convention, see the [design token naming guidelines](../../primitives/naming-convention) page.
+
+<img
+  width="960"
+  alt="wip"
+  src="https://github.com/primer/primitives/assets/18661030/10dd4c91-5faa-464c-ab07-9e26afd54e5f"
+  style="margin-bottom: 1rem;"
+/>
+
+**Base** color tokens are the lowest level tokens and map directly to a raw value. They are <strong>only</strong> to be used as a reference for functional and component/pattern tokens. Base color tokens don't respect color modes and should never be used directly in code or design.
+
+**Functional** color tokens represent global UI patterns such as text, borders, shadows, and backgrounds. These are the most commonly used design tokens throughout all of Primer and GitHub UI. Functional color tokens reference base color tokens under the hood, and respect color modes.
+
+<img
+  width="960"
+  alt="wip"
+  src="https://github.com/primer/react/assets/18661030/200cd7f0-91dd-4e4e-b574-26612bc191b8"
+  style="margin-bottom: 1rem; margin-top: 1rem;"
+/>
```

Comment:
Nitpick: I wonder if in this image, we could add the same purple text label that was used in the previous example and show **Component/pattern**

Reply:
Both examples are the same type, does this image still read that way? Great suggestion.

<img width="960" alt="pattern color example" src="https://github.com/primer/design/assets/18661030/3481d69b-f121-444b-ade6-e560faf423b1">